README.md

```diff
@@ -2,11 +2,17 @@
   <img src="https://github.com/box/sdks/blob/master/images/box-dev-logo.png" alt= “box-dev-logo” width="30%" height="50%">
 </p>
 
+# Deprecation notice
+
+This version of the Box Python SDK is under maintenance mode, and will be deprecated soon, only critical security updates and bug fixes will be provided. We recommend using the new version Box Python SDK, which can be found at [box/box-python-sdk-gen](https://github.com/box/box-python-sdk-gen)
+
+From Box Python SDK v3.x, you can find the migration guide [here](https://github.com/box/box-python-sdk-gen/blob/main/migration-guide.md). If you have any questions, please create an issue in the new repository or reach out to [Box Developer Support](https://developer.box.com/support/).
```

Comment:
You can find the migration guide here for transitioning from Box Python SDK v3.x to the new python-sdk-gen package.